armotypes/vulnerabilitytypes.go

```diff
@@ -9,11 +20,100 @@ type VulnerabilityJobParams struct {
 	JobID           string `json:"jobID,omitempty"`
 }
 
+type VulnerabilityWorkload struct {
+	Wlid             string                     `json:"wlid"`
+	Name             string                     `json:"name"`
+	Namespace        string                     `json:"namespace"`
+	Kind             string                     `json:"kind"`
+	ClusterName      string                     `json:"clusterName"`
+	ClusterShortName string                     `json:"clusterShortName"`
+	CustomerGUID     string                     `json:"customerGUID"`
+	ImagesCount      int                        `json:"imagesCount"`
+	SeverityStats    map[string]SeverityDetails `json:"severityStats"`
+	RiskFactors      []RiskFactor               `json:"riskFactors"`
+	Labels           []string                   `json:"labels"`
+}
+
+type SeverityDetails struct {
+	Total   int             `json:"total"`
+	TopCves []TopCveDetails `json:"topCves"`
+}
+
+type TopCveDetails struct {
+	Name string `json:"name"`
+}
+
+type ImagePathInfo struct {
+	WorkloadId       string `json:"wlid"`
```

Comment:
no wlid in container scan it is only for posture 
We have workload hash but it is just combination of cluster, namespace ,kind, name, container-name